### PR TITLE
Plone5 integration as pattern

### PR DESCRIPTION
**Pull request by petschki**
- migrate resources to compiled Plone5 bundle.
- change behaviour as a ``drop-in`` replacement for ``plone.mainnavigation``

Changes are targeting Plone5 *only* so this pullrequest is an open discussion ... input and issues welcome.

If these changes are approved and released, the current master has to move to something like ``1.x`` branch

### CODE REVIEW

File: collective/superfish/upgrades/__init__.py [hunk 1]
```diff
@@ -14,3 +13,8 @@ def _cookResources(portal):
 def to_2(context):
     portal = getToolByName(context, 'portal_url').getPortalObject()
     _cookResources(portal)
+
+
+def to_5(context):
+    setup = getToolByName(context, 'portal_setup')
+    setup.runImportStepFromProfile(PROFILE_ID, 'plone.app.registry')
```
Comment (frisi) [inline collective/superfish/upgrades/__init__.py]: i personally would copy current registry.xml to the 2_to_5 upgrade profile folder and skip this method
this way the registry settings done in this upgrade step will stay the same. even if registry.xml gets changed in future releases

**Comment by petschki** [comment]
Well regarding the Plone 5 resources I took this addon as a kick in my butt to wrap my head around... still not 100% sure, if this is the way to go... but it works 🤗 ... maybe @thet or @frapell are willing to give us some more feedback/insights on this? Basically I followed the examples which we made in francos training at the barcelona conference ... one thing I'd like to know is, if we need a separate bundle for this addon (which is an extra request in the browser) or if there's a way to merge this into the plone bundle?

**Comment by frapell** [comment]
It looks ok for the most part, however I do have one suggestion. I'm noticing that on https://github.com/collective/collective.superfish/pull/5/files#diff-602aa72afd2fe2f4d36cdc3fe312ab3d you do ```$('ul.sf-menu').superfish();```. Instead, I would implement this as a pattern, so you can just assign ```class="pat-superfish"``` to any ```<ul>``` which will trigger the pattern. This way you can also include ```data-pat-superfish``` on each of these elements, to pass different configurations.

**Comment by petschki** [comment]
@frapell yes you're right. This way it's much easier to inject the available options to `$.superfish()`

**Comment by petschki** [comment]
@frisi I've created the `1.x` branch from current master and merged this pull request. I think we can go ahead and release this, to get some feedback.

**Comment by frisi** [comment]
of course - please go ahead! ;-)

**Pull request merged**